OpenRA.Mods.Common/Traits/Buildings/ProductionAirdrop.cs

```diff
@@ -21,7 +21,10 @@ namespace OpenRA.Mods.Common.Traits
 	public class ProductionAirdropInfo : ProductionInfo
 	{
 		[NotificationReference("Speech")]
-		public readonly string ReadyAudio = "Reinforce";
+		[Desc("Speech notification to play when a unit is delivered.")]
+		public readonly string ReadyNotification = "Reinforce";
```

Comment:
This would need an update rule.

Reply:
I renamed this field to match the style that is used in most places ("Audio" vs "Notification"). However there are also several other places that use "Audio" or "Sound". This is an opportunity to unify the terms and the language across all traits that implement text/speech notifications. Should I go for that or is it too much work for little gain? :thinking:

Reply:
I'm willing to do it but let's defer to a follow up. I will undo the rename here.